nucypher/cli/painting.py

```diff
@@ -247,14 +249,14 @@ def paint_staking_confirmation(emitter, ursula, transactions):
 
 def prettify_stake(stake, index: int = None) -> str:
 
-    start_datetime = stake.start_datetime.local_datetime().strftime("%b %d %H:%M:%S %Z")
-    expiration_datetime = stake.end_datetime.local_datetime().strftime("%b %d %H:%M:%S %Z")
+    start_datetime = str(stake.start_datetime.slang_date())
```

Comment:
I thought we didn't want to use`slang_date()` for stake enactment time. Was this on purpose, or a possible rebase that overwrote a change.

Reply:
What is the *disadvantage* of slang time here?

Reply:
See PR #1211 and Issue #1198.

Reply:
Slang time would be OK if midnights were at the same time for days and for periods.